src/v/pandaproxy/schema_registry/sharded_store.cc

```diff
@@ -268,21 +265,20 @@ sharded_store::get_subjects(include_deleted inc_del) {
 }
 
 ss::future<std::vector<schema_version>>
-sharded_store::get_versions(const subject& sub, include_deleted inc_del) {
+sharded_store::get_versions(subject sub, include_deleted inc_del) {
     co_return co_await _store.invoke_on(
       shard_for(sub), _smp_opts, [sub, inc_del](store& s) {
           return s.get_versions(sub, inc_del).value();
       });
 }
 
-ss::future<bool>
-sharded_store::is_referenced(const subject& sub, schema_version ver) {
-    auto map = [&sub, ver](store& s) { return s.is_referenced(sub, ver); };
+ss::future<bool> sharded_store::is_referenced(subject sub, schema_version ver) {
+    auto map = [sub, ver](store& s) { return s.is_referenced(sub, ver); };
```

Comment:
move into the capture to avoid the copy? (I wonder if 03 sometimes does this start of thing. sure would be nice).